### PR TITLE
Added template outputs field to stack

### PR DESCRIPTION
**Pull request by jkorkalainen**
It would be usable to include Heat template outputs as part of stack object.

### CODE REVIEW

**Comment by kenperkins** [comment]
I'm confused why this is failing on Travis. I get errors I don't see on local.

**Comment by kenperkins** [comment]
OK, I've narrowed it down to a `jshint` behavior change. Researching.

**Comment by kenperkins** [comment]
https://github.com/jshint/jshint/issues/2229

File: lib/pkgcloud/openstack/orchestration/stack.js [hunk 1]
```diff
@@ -45,15 +45,12 @@ Stack.prototype._setProperties = function (details) {
   this.createdAt = details['creation_time'];
   this.updatedAt = details['updated_time'];
 
+  this.outputs = details['outputs'];
```
Comment (kenperkins) [inline lib/pkgcloud/openstack/orchestration/stack.js]: I'm not finding documentation for this parameter. Can you point me to it?

Comment (jkorkalainen) [inline lib/pkgcloud/openstack/orchestration/stack.js]: http://docs.openstack.org/developer/heat/template_guide/hot_spec.html#outputs-section

Comment (kenperkins) [inline lib/pkgcloud/openstack/orchestration/stack.js]: Perfect, thanks. So it doesn't appear to come back from `createStack` but rather `getStack`?

Comment (jkorkalainen) [inline lib/pkgcloud/openstack/orchestration/stack.js]: Yes, that's right.

**Comment by kenperkins** [comment]
Good idea. Perhaps we should document the model for `getStack` in https://github.com/pkgcloud/pkgcloud/blob/master/docs/providers/openstack/orchestration.md

**Comment by jkorkalainen** [comment]
Documentation is a good idea. Or at least a link to that HOT specification: docs.openstack.org/developer/heat/template_guide/hot_spec.html#, which explains the functionality.

**Pull request merged**